### PR TITLE
Enable rln-relay on waku v2 test fleets

### PR DESCRIPTION
**Pull request by staheri14**

### CODE REVIEW

**Comment by jakubgs** [comment]
You will need to add support for these variables in the role: https://github.com/status-im/infra-role-nim-waku

File: ansible/group_vars/wakuv2-prod.yml [hunk 1]
```diff
@@ -32,6 +32,9 @@ nim_waku_websockify_ssl_dir: '{{ nim_waku_websocket_ssl_dir }}'
 nim_waku_websockify_ssl_cert: '{{ nim_waku_websocket_ssl_cert }}'
 nim_waku_websockify_ssl_key: '{{ nim_waku_websocket_ssl_key }}'
 
+# Waku rln-relay parameters
+nim_waku_rln_relay_content_topic: '/toy-chat/2/luzhou/proto'
```
Comment (jakubgs) [inline ansible/group_vars/wakuv2-prod.yml:36]: What's the point of this setting it for prod if `rln-relay` is not enabled for it?

Comment (staheri14) [reply ansible/group_vars/wakuv2-prod.yml:36]: I think with. my recent change and checking whether rln-relay is enabled in the protocols set, then I can safely remove this variable from waku v2 prod variables.

**Comment by jakubgs** [comment]
Why did you merge this before you merged the role changes?

**Comment by staheri14** [comment]
I thought this will make the result I was looking for (could not see any issue):
What I needed is to enable rln-relay on test fleets on the following content topic '/toy-chat/2/luzhou/proto', which is exactly what should have happened after merging this PR:
- The protocol is added to the set of enabled protocols for test fleets in this PR. 
- The docker-compose template will enable it on test fleets due to https://github.com/status-im/infra-role-nim-waku/blob/1e5367c86a906d9b08c0d492f5d70a85d790ae37/templates/docker-compose.yml.j2#L40
- And it will use the default rln-relay content topic due to https://github.com/status-im/nim-waku/blob/54d64a63fed236bd33667db225a3a4f6cccdb71c/waku/v2/node/config.nim#L114 
Am I missing something?

**Comment by staheri14** [comment]
@jakubgs The reason that I opened this infra-role PR https://github.com/status-im/infra-role-nim-waku/pull/1/files was to make the content topic more explicit, so that future updates on https://github.com/status-im/nim-waku/blob/54d64a63fed236bd33667db225a3a4f6cccdb71c/waku/v2/node/config.nim#L114 won't affect the test feets.

**Comment by jakubgs** [comment]
But without that PR merged the `nim_waku_rln_relay_content_topic` variable has no effect.

**Comment by jakubgs** [comment]
Deployed:
```
 > a all -a 'grep rln /docker/nim-waku-v2/docker-compose.yml'
node-01.do-ams3.wakuv2.test | CHANGED | rc=0 >>
      --rln-relay=true
      --rln-relay-content-topic='/toy-chat/2/luzhou/proto'
node-01.gc-us-central1-a.wakuv2.test | CHANGED | rc=0 >>
      --rln-relay=true
      --rln-relay-content-topic='/toy-chat/2/luzhou/proto'
node-01.ac-cn-hongkong-c.wakuv2.test | CHANGED | rc=0 >>
      --rln-relay=true
      --rln-relay-content-topic=/toy-chat/2/luzhou/proto
```

**Pull request merged**